pymks/fmks/correlations.py

```diff
@@ -0,0 +1,87 @@
+"""MKS Correlation Module
+
+For computing auto and cross corelations under assumption
+of periodic boundary conditions using discreete fourier
+transform.
+"""
+
+
+import numpy as np
+from toolz.curried import pipe, curry
+from .func import dafftshift, dafftn, daifftn, daconj
+
+
+def faxes(arr):
```

Comment:
This is not the same as https://github.com/materialsinnovation/pymks/blob/develop/pymks/fmks/localization.py#L113? Isn't the auto and cross correlation done after the discretization? Shouldn't the last axis be ommitted?

Reply:
Yes, because all the discretized states are note passed to correlation module - only the pair for which statistics is to be computed.